src/sensors/distance/DistanceSensor.hpp

```diff
@@ -4,7 +4,7 @@
  */
 #pragma once
 
-#include <stdint.h> // NOLINT(modernize-deprecated-headers)
+#include <stdint.h>
```

Comment:
:warning: **modernize-deprecated-headers** :warning:
inclusion of deprecated C++ header 'stdint.h'; consider using 'cstdint' instead
```suggestion
#include <cstdint>

```